File: VB/10_Convertors/ToPDF.vb
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms

Public Class Form1

    Private Sub btnRun_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnRun.Click

        'create PPT document
        Dim presentation As New Presentation()

        'load PPT file from disk
        presentation.LoadFromFile("..\..\..\..\..\..\Data\Presentation1.pptx")

        'save the PPT do PDF file format
        presentation.SaveToFile("ToPdf.pdf", FileFormat.PDF)
        System.Diagnostics.Process.Start("ToPdf.pdf")

    End Sub
End Class

File: CS/10_Convertors/ToPDF.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace Spire.Presentation.Demo
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            //create PPT document
            Presentation presentation = new Presentation();

            //load PPT file from disk
            presentation.LoadFromFile(@"..\..\..\..\..\..\Data\Presentation1.pptx");

            //save the PPT do PDF file format
            presentation.SaveToFile("ToPdf.pdf", FileFormat.PDF);
            System.Diagnostics.Process.Start("ToPdf.pdf");

        }
    }
}